### PR TITLE
Dispatch upgradeAppStoreApps event

### PR DESCRIPTION
**Pull request by VicDeo**

### CODE REVIEW

File: lib/private/Updater.php [hunk 1]
```diff
@@ -386,23 +388,17 @@ private function isCodeUpgrade() {
...
-		//
-		// TODO: integrate market app in here
-		//
-
-//		foreach($disabledApps as $app) {
-//			try {
-//				if (Installer::isUpdateAvailable($app)) {
-//					$ocsId = \OC::$server->getConfig()->getAppValue($app, 'ocsid', '');
-//
-//					$this->emit('\OC\Updater', 'upgradeAppStoreApp', [$app]);
```
Comment (PVince81) [inline lib/private/Updater.php]: keep old events for backward compatibility ?

**Comment by PVince81** [comment]
apart from that looks good to me.

still WIP ?

File: lib/private/Updater.php (hunk 1 above)
Comment (VicDeo) [inline lib/private/Updater.php]: @PVince81 it was not me, who commented that out. IIRC it was commented when app management part was stripped from core

**Comment by VicDeo** [comment]
@PVince81 I haven't tested it yet. I'm not used to supply untested code

**Comment by PVince81** [comment]
would still be good to keep the old hook, unless you think it's private and not used anywhere.

if that's the case, then 👍

**Comment by VicDeo** [comment]
@PVince81 It has a listener in `core/Command/Upgrade.php` so I put it back

**Pull request merged**